eZ/Bundle/EzPublishCoreBundle/Resources/config/slot.yml

```diff
@@ -15,6 +17,7 @@ parameters:
     ezpublish.http_cache.signalslot.unhide_location.class: eZ\Publish\Core\MVC\Symfony\Cache\Http\SignalSlot\UnhideLocationSlot
     ezpublish.http_cache.signalslot.update_location.class: eZ\Publish\Core\MVC\Symfony\Cache\Http\SignalSlot\UpdateLocationSlot
     ezpublish.http_cache.signalslot.update_user.class: eZ\Publish\Core\MVC\Symfony\Cache\Http\SignalSlot\UpdateUserSlot
+    ezpublish.http_cache.signalslot.update_user_group.class: eZ\Publish\Core\MVC\Symfony\Cache\Http\SignalSlot\UpdateUserSlot
```

Comment:
If it is the same I guess we can omit this or?

Reply:
Nice catch! It shouldn't be the same, because there is not `userId` property in `\eZ\Publish\Core\SignalSlot\Signal\UserService\CreateUserGroupSignal` :wink: